portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase

```diff
@@ -612,7 +612,6 @@ definition {
 	test CanBeValidatedUsingTheDoesNotContainOption {
 		property portal.acceptance = "true";
 		property portal.upstream = "true";
-		property test.name.skip.portal.instance = "FormsTextField#CanBeValidatedUsingTheDoesNotContainOption";
```

Comment:
@leticiachiu, why did you decide to remove it?

Reply:
Because didn't see why is necessary.. it is? if so, I cant put it back :)

Reply:
Yes, it is. Please check this [Slack thread](https://liferay.slack.com/archives/CL8DPKWPN/p1598361447009100). :wink: